### ISSUE TITLE
feature request: View Trace button

### ISSUE THREAD

**Issue by connorjclark** [GoogleChrome/lighthouse-ci#112]
We could collect the traces and show a View Trace button in the compare view (or, maybe in core allow a way to view the trace).

Can leverage the new tracing app (https://docs.perfetto.dev/#/) I suspect it should have a way to postMessage a trace file.

This came up when demo'ing LHCI to a user - was a big want.

**Comment by patrickhulce** [comment]
Agreed this would be *awesome*, also trace diffing UI would be hella dope :)

Saving the traces could definitely get difficult, but minified could work... we'll see how to fit this in
